src/nrniv/netpar.cpp

```diff
@@ -123,22 +117,25 @@ int ncs_netcon_mindelays( int**hosts, double **delays )
 
 double ncs_netcon_localmindelay( int srcgid )
 {
-    PreSyn *ps;
-    gid2out_->find( srcgid, ps );
-    assert(ps);
-    
+    PreSyn *ps = gid2out_.at[srcgid];
```

Comment:
```suggestion
    PreSyn *ps = gid2out_.at(srcgid);
```
assuming you want it to throw if the given gid is not present.